ui/packages/catalog-ui-search/src/main/webapp/component/query-custom/query-custom.view.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define, setTimeout*/
+define([
+    'marionette',
+    'underscore',
+    'jquery',
+    './query-custom.hbs',
+    'js/CustomElements',
+    'component/filter-builder-search-form/filter-builder.view',
+    'component/filter-builder/filter-builder',
```

Comment:
❓ Wait why is the view coming from `filter-builder-search-form` and the model from `filter-builder`? Where does `component/filter-builder/filter-builder/view.js` factor into this?

Reply:
The model need not be extended; the views use the same model but are presented a little differently.  `component/filter-builder/filter-builder.view.js` is used in the `query-advanced` view